### PR TITLE
submitting Lesson08

### PR DESCRIPTION
**Pull request by Reem0330**
Thank you for your quick response, removing the docstring fixed the issue..

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
Closing this PR - please update your code and then open a new PR once corrected. 

(your code still has the syntax error)

**Comment by Reem0330** [comment]
can I get a hint on where is the syntax error please? I have pytest 20 test
passed ..

Thank you
Reem

File: students/ReemAlqaysi/Lesson08/circle.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+"""Step 4:
+Add an area property so the user can get the area of the circle:"""
+    @property
+    def area(self):
+        return math.pow(self.radius, 2)*math.pi
+
+"""Step 5:
+Add an “alternate constructor” that lets the user create a Circle directly with the diameter:"""
+    def from_diameter(self,diameter):
+    	return self.radius = diameter/2
```
Comment (natasha-aleksandrova) [inline students/ReemAlqaysi/Lesson08/circle.py]: I commented on this line in the last PR - this line is a syntax error. you cannot have a return and assignment in the same line, try this code in the python shell.

**Comment by Reem0330** [comment]
I'm having issues with git in somehow bellow is the old script when I had
an issue, and it's fixed now, without knowing about the return and the
assignment, roles I fixed it already in the code, I'll try to submit the
assignment for lesson08 again, for step 5 I have it as below now..thank you
so much

    @classmethod
    def from_diameter(cls, diameter):
        radius = diameter/2
        return cls(radius)

**Comment by Reem0330** [comment]
I'm having issues submitting the both assignments for lesson 08 and 09 ...
and conversation in git in somehow going back to the previous submits..
how I can organize or start a new pull request without going back or merge
with the old submit?
since I'm have less than two days left for my class end date, I'm
attaching  the both assignments here in zip files in case you can't see
those files .. because I can't see them still in git..

please advice
Thank you
Reem Alqaysi

**Pull request closed without merging**